### PR TITLE
Add automation test for mri_violations

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

File: modules/mri_violations/test/mri_violationsTest.php [hunk 1]
```diff
@@ -52,7 +200,231 @@ function testMriProtocolCheckViolationsDoesPageLoad()
...
+        $resolutionElement = $this->safeFindElement(WebDriverBy::Name
+               ("resolvable[c57b919a921eaa1a43bb5e0c44cd4226]"));
+        $resolution = new WebDriverSelect($resolutionElement);
+
+        $value = $resolution->getFirstSelectedOption()->getAttribute('value');
+        $this->assertEquals("inserted",$value);
+    }
+}
+?>
+                                                                                                                                                                                    428,1         Bot
```
Comment (driusan) [inline modules/mri_violations/test/mri_violationsTest.php]: It looks like there's an extra line at the bottom that says "428,1         Bot". I don't think this is supposed to be here.

File: modules/mri_violations/test/mri_violationsTest.php [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (driusan) [inline modules/mri_violations/test/mri_violationsTest.php]: I don't think this blank line is supposed to be here. It's not much of an issue for tests, but blank lines can confuse/break PHP

**Comment by driusan** [comment]
Let me know when the line issues at the start/end are fixed and I'll tag this as passed.

File: modules/mri_violations/test/mri_violationsTest.php (hunk 1 above)
Comment (kongtiaowang) [inline modules/mri_violations/test/mri_violationsTest.php]: @driusan         I will remove the extra line.

**Comment by kongtiaowang** [comment]
@driusan  
I removed the blank line and bottom line. pls add the passcode tag.

**Pull request merged**